### PR TITLE
feat: add support for markdown rendering

### PR DESCRIPTION
**Pull request by KnisterPeter**
To simplify content management there is a building helper 'markdown' and a buildin template
'Markdown' which simply render a text file to react.

ISSUES CLOSED: #17

### CODE REVIEW

File: src/helpers.js [hunk 1]
```diff
@@ -4,12 +4,15 @@ const requireAll = require('require-all');
...
 
+const {renderMarkdownFile} = require('./markdown');
+
 module.exports = {
 	loadHelpers
 };
 
 const TICK = chalk.bold.green(figures.tick);
 const CROSS = chalk.bold.red(figures.cross);
+const WARN = chalk.bold.red(figures.warning);
```
Comment (marionebl) [inline src/helpers.js]: Perhaps yellow would make it easier to distinguish from errors.

File: src/helpers.js [hunk 2]
```diff
@@ -20,6 +23,11 @@ function loadHelpers(srcHelpers) {
 		dirname: path.resolve(srcHelpers),
 		map: name => name.replace(/-([a-z])/g, (m, c) => c.toUpperCase())
 	});
+	if (typeof helpers.markdown === 'undefined') {
```
Comment (marionebl) [inline src/helpers.js]: I think this is mildly surprising - do you want to allow custom implementations here? I'd expect schlump to plainly overwrite what it needs in the helpers namespace and log a warning about that fact. Perhaps the message is more easily phrased then to, the current one is a bit ambiguous.

File: src/helpers.js (hunk 1 above)
Comment (KnisterPeter) [inline src/helpers.js]: Yellow is very bad readable on many displays. There are other places in schlump which also use red for warnings.

Comment (marionebl) [inline src/helpers.js]: Sold

File: src/helpers.js (hunk 2 above)
Comment (KnisterPeter) [inline src/helpers.js]: If ever there is a reason to create custom markdown configuration (like rendering <p class="some">foo</p>) then there is the need to overwrite it.
Schlump only brings a default markdown handler and is not configurable in this.

Comment (KnisterPeter) [inline src/helpers.js]: You don't enable the default one. Markdown is enabled by default for convenience (as in almost any static site generator)

**Pull request merged**